CONTRIBUTING.md

```diff
@@ -1,6 +1,29 @@
-Before contributing code, please note:
+# Before reporting a bug
+1. Search the [issue tracker](https://github.com/aframevr/aframe/issues) for similar issues. 
+2. Try the latest dev branch version of [aframe](https://github.com/aframevr/aframe/tree/dev).
```

Comment:
I don't think we need people to check out the dev branch before reporting an issue.